packages/plugins/sentry/README.md

```diff
@@ -37,8 +37,6 @@ yarn add @sentry/node @sentry/tracing @envelop/sentry
 import { execute, parse, specifiedRules, subscribe, validate } from 'graphql'
 import { envelop, useEngine } from '@envelop/core'
 import { useSentry } from '@envelop/sentry'
-// do this only once in you entry file.
-import '@sentry/tracing'
```

Comment:
Is this not needed anymore?

Reply:
Nope, as I wrote in the PR description : 
>Since Sentry SDK v10, you don’t need to import '@sentry/tracing' anymore.
> Removed the imports in documentation and test, already embedded in sentry/node or sentry/nextjs.

I can try to retrieve some sentry migration link if you want.